integration_tests/dbt_project.yml

```diff
@@ -2,7 +2,7 @@ name: 'integration_tests'
 version: '1.0.0'
 config-version: 2
 
-profile: default
```

Comment:
@kuljisd same issue with this one. Local testing config shouldn't be pushed in a branch.